pkg/controller/storagecluster/reconcile.go

```diff
@@ -903,6 +904,22 @@ func (r *ReconcileStorageCluster) deleteResources(sc *ocsv1.StorageCluster, reqL
 	return r.deleteNoobaaSystems(sc, reqLogger)
 }
 
+//getCephDaemonPlacements returns placement configuration for ceph components with appropriate topology
+func getCephDaemonPlacements(sc *ocsv1.StorageCluster, component string) rook.Placement {
```

Comment:
This could be called once for `all`, saving extra calls to each individual component. All of them have the same placement ultimately.

Reply:
Have some doubts around this. Please see https://github.com/openshift/ocs-operator/pull/405/files#r381838850